.eslintrc

```diff
@@ -1,8 +1,12 @@
 {
   "env": {
-    "jest/globals": true
+    "jest/globals": true,
   },
-  "extends": ["airbnb","react-app", "plugin:jest-dom/recommended", "plugin:testing-library/react"],
+  "parser": '@babel/eslint-parser',
+  "parserOptions": {
+    sourceType: 'module',
+  },
+  "extends": ["airbnb","plugin:react/recommended", "plugin:jest-dom/recommended", "plugin:testing-library/react"],
```

Comment:
Do we also need `"plugin:react-hooks/recommended"`? It seems like create-react-app also gave us `'react-hooks/rules-of-hooks': 'error'`, at least.

I think CRA also gave us flowtype, but it doesn't look like we have any flowtype comments to lint, so maybe that's fine?

Reply:
Sure I've added "plugin:react-hooks/recommended"